tokenserver-settings/src/lib.rs

```diff
@@ -78,6 +80,7 @@ impl Default for Settings {
             database_pool_max_size: 10,
             database_pool_min_idle: None,
             database_pool_connection_timeout: Some(30),
+            database_request_timeout: Some(30),
```

Comment:
I'd like to have the option to completely disable the timeouts and any potential tokio timer overhead along with it -- i.e. when set to disabled, skip creating the timeout entirely.

See deadpool's example along these lines: [deadpool::managed::apply_timeout](https://github.com/bikeshedder/deadpool/blob/2a584a99fa6d80921f6428b7c5e7e6dc06b8e0a9/src/managed/mod.rs#L648)

Reply:
Fair.

Reply:
Due to that config issue I mentioned the other day, I think we just need to set this to None and apply a value in the cloudops config, otherwise there's no way to force it back to None

```suggestion
            database_request_timeout: None,
```